examples/01-filter/image-representations.py

```diff
@@ -0,0 +1,223 @@
+# before plotting.
+
+points_ithresh_as_cells = points_ithresh.points_to_cells()
+points_ithresh_as_cells.plot(show_edges=True)
+
+################################################################################
+# The binary data is now correctly visualized as binary data.
+
+################################################################################
+# Cell Filters with Image Data
+# ----------------------------
+# Use a cell representation of the image when working with cell-based filters
+# such as :meth:`~pyvista.DataSetFilters.threshold`. If the image only has point
+# data, use :meth:`~pyvista.ImageDataFilters.points_to_cells` to re-mesh the
+# input first. Here, we reuse the cell-based image created earlier.
+#
+# For context, we first show the input data array.
+
+cells_volume.cell_data['Data']
+
+################################################################################
+# Now apply the filter and print the result.
+
+cells_thresh = cells_volume.threshold(2)
+cells_thresh.cell_data['Data']
+
+################################################################################
+# When the input is cell data, this filter returns seven discrete values greater
+# than or equal to the threshold value of ``2`` as expected.
+#
+# If we plot the result, the cells also produce the correct visualization.
+
+cells_thresh.plot(show_edges=True)
+
+################################################################################
+# However, if we apply the same filter to a point-based representation of the
+# image, the filter does not produce the desired result.
+
+points_thresh = points_volume.threshold(2)
+points_thresh.point_data['Data']
+
+################################################################################
+# In this case, since the points image only has a single cell, the filter has no
+# effect on the data array's values. The thresholded values are the same as the
+# input values.
+#
+# Plotting the result confirms this. The plot is identical to the initial plot
+# of the point-based image shown at the beginning of this example.
+
+points_thresh.plot(show_edges=True)
+
+################################################################################
+# Representations of 2D Images
+# ----------------------------
+# The filters :meth:`~pyvista.ImageDataFilters.points_to_cells` and
+# :meth:`~pyvista.ImageDataFilters.cells_to_points` can similarly be used
+# with 2D images.
+#
+# For this example, we create a 4x4 2D grayscale image with 16 points to represent
+# 16 pixels.
+
+data_array = np.linspace(0, 255, 16, dtype=np.uint8)[::-1]
+points_image = pv.ImageData(dimensions=(4, 4, 1))
+points_image.point_data['Data'] = data_array
+
+################################################################################
+# Plot the image. As before, the plot does not appear correct since the point
+# data is interpolated and nine cells are shown rather than the desired 16
+# (one for each pixel).
```

Comment:
```suggestion
# Plot the image. As before, the plot does not appear correct since the point
# data is interpolated, and nine cells are shown rather than the desired 16
# (one for each pixel).
```